tests/config.py

```diff
@@ -0,0 +1,30 @@
+# coding=utf-8
+
+"""
+Variable File for VIC-UI Robot tests
+
+Usage:
+robot --variablefile config.py my_robot_test.robot
+
+See:
+http://robotframework.org/robotframework/latest/RobotFrameworkUserGuide.html#resource-and-variable-files
+"""
+
+# TODO add support for reading OS env vars and/or other sources
+# TODO improve naming
+# TODO use structures
+
+MACOS_HOST_IP = '10.25.201.232'
+UBUNTU_HOST_IP = '10.20.121.145'
+WINDOWS_HOST_IP = '10.25.200.225'
+# TODO delete/unused BUILD_3620759_IP = '10.25.200.237'
```

Comment:
This should not be deleted, or it will cause tests to fail.

Reply:
Coundn't find where these vars are used, only it's definition. 
Also seening that the var name is tight to a particular build number, I suspected it was for a temporary use/test and not for production.

Are you sure these vars are still in use ?